### ISSUE TITLE
UnicodeEncodeError when importing SLES11SP4 repo

### ISSUE THREAD

**Issue by matya** [uyuni-project/uyuni#621]
I have tried to import the regular SLES11 SP4 repo to a repository, and I got this backtrace.
```
2019/02/22 15:39:55 +02:00   Importing packages to DB:
2019/02/22 15:41:25 +02:00 20.6 %
2019/02/22 15:42:55 +02:00 32.37 %
2019/02/22 15:44:25 +02:00 33.64 %
2019/02/22 15:45:55 +02:00 72.43 %
2019/02/22 15:47:26 +02:00 97.84 %
2019/02/22 15:47:33 +02:00 Importing packages finished.
2019/02/22 15:47:33 +02:00 
2019/02/22 15:47:33 +02:00   Linking packages to the channel.
2019/02/22 15:47:38 +02:00 
2019/02/22 15:47:38 +02:00   Patches in repo: 0.
2019/02/22 15:47:38 +02:00 Unexpected error: <type 'exceptions.UnicodeEncodeError'>
2019/02/22 15:47:38 +02:00 Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/spacewalk/satellite_tools/reposync.py", line 581, in sync
    self.import_products(plugin)
  File "/usr/lib/python2.7/site-packages/spacewalk/satellite_tools/reposync.py", line 1875, in import_products
    products = repo.get_products()
  File "/usr/lib/python2.7/site-packages/spacewalk/satellite_tools/repo_plugins/yum_src.py", line 572, in get_products
    p['description'] = _fix_encoding(product.find('description').text)
  File "/usr/lib/python2.7/site-packages/spacewalk/satellite_tools/repo_plugins/yum_src.py", line 933, in _fix_encoding
    return str(text)
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2019' in position 204: ordinal not in range(128)
```

Versions:
```
# rpm -qa | grep uyu
uyuni-build-keys-web-12.0.1-4.1.noarch
uyuni-build-keys-12.0.1-4.1.noarch
patterns-uyuni_server-4.0-4.1.x86_64
release-notes-uyuni-4.0.1-2.1.x86_64
```

**Comment by brejoc** [comment]
@dincamihai @meaksh: Didn't we have fix for this? Do guys still remember?

**Comment by juliogonzalez** [comment]
Ping @dincamihai, @meaksh

**Comment by brejoc** [comment]
Related PR: https://github.com/uyuni-project/uyuni/pull/404

Just like #404 I'm closing this issue, because it will be fixed with the switch to Python3 in 4.0.2, which will probably be released around April.

Thanks a lot for the report! Please free to re-open if you think we should keep it.
